test/e2e/autoscaling/horizontal_pod_autoscaling.go

```diff
@@ -106,18 +110,50 @@ var _ = SIGDescribe("[Feature:HPA] Horizontal pod autoscaling (scale resource: C
 	})
 })
 
+var _ = SIGDescribe("[Feature:HPA] Horizontal pod autoscaling (scale resource: custom metrics)", func() {
+	f := framework.NewDefaultFramework("horizontal-pod-autoscaling")
+	f.NamespacePodSecurityEnforceLevel = api.LevelBaseline
+
+	// Set up FeatureGate for enabling HPAScaleToZero, which is hidden by alpha feature gate.
+	fg := featuregate.NewFeatureGate()
+	fg.Add(map[featuregate.Feature]featuregate.FeatureSpec{"HPAScaleToZero": {
+		Default:       false,
+		LockToDefault: false,
+		PreRelease:    featuregate.Alpha,
+	}})
+	e2eskipper.InitFeatureGates(fg, nil)
+
+	ginkgo.Describe("[Serial] [Slow] Scale down to zero", func() {
```

Comment:
Is there a naming convention for e2e tests?
Other tests:
- use controller type and then "should scale..."
- add [feature:<name>] at the end, so I would expect [Feature:HPAScaleToZero] at the end

Reply:
I fixed the 1st one. Regarding 2nd one (Feature:HPAScaleToZero) - I think we keep Feature:HPA in all HPA regarding tests, including Behavior one.